mslib/utils/airdata.py

```diff
@@ -252,7 +252,7 @@ def get_airspaces(countries=None):
                                             for data in airspace_data["polygon"].split(",")]
                 _airspaces.append(airspace_data)
                 _airspaces_mtime[file] = os.path.getmtime(os.path.join(OSDIR, "downloads", "aip", file))
-            else:
-                QtWidgets.QMessageBox.information(None, "No Airspaces data in file:", f"{file}")
+        else:
```

Comment:
this prior indenting was correct. The reason is to show to the user that the requested files has no data and is therefore not shown.

we use seldom the for/else clause,  https://book.pythontips.com/en/latest/for_-_else.html

Without this there is no messageBox shown like 
![hint](https://github.com/Open-MSS/MSS/assets/278728/887e1c9d-fec7-456e-a5ba-9a15daf7da19)

This is not that comfortable for a user, but when this gets improved it maybe should not done in this PR?

Reply:
should have looked into the file downloaded. 

110238 Sep  1 10:36 ae_asp.xml

I agree to the change.

Reply:
later we need some kind of feedback to the user, that his selection is not what he gets shown on the map.